packages/RichTextEditing-Core.package/RichTextDocument.class/instance/emptyChars.st

```diff
@@ -0,0 +1,4 @@
+count
+emptyChars
+
+	^ #(182 32 10 13)
```

Comment:
what about a method:
``` 
spaceValue 
^32 
```
as in RichTextEditorModel?

Reply:
what do you mean? do you want the method to be more specific?

Reply:
Yea, or maybe a comment stating what each char is

Reply:
> 
> 
> Yea, or maybe a comment stating what each char is

implemented in 
350982f